code/_onclick/click.dm

```diff
@@ -19,6 +19,7 @@
 /atom/Click(location, control, params)
 	if(flags_atom & INITIALIZED)
 		SEND_SIGNAL(src, COMSIG_CLICK, location, control, params, usr)
+		SEND_SIGNAL(usr, COMSIG_DIRECT_ATTACK, location, control, params, src, usr)
```

Comment:
Why do you have a new signal? How is this different from `COMSIG_CLICK`

Reply:
I was told to do this by Hughgent